src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/FileUploadWithTagComponent.tsx

```diff
@@ -0,0 +1,812 @@
+    marginTop: '2.4rem',
+    tableLayout: 'fixed',
+    marginBottom: '12px',
+    width: '100%',
+    display: 'grid',
+    gridTemplateColumns: '5fr 3fr 4fr',
+    '& thead': {
+      display: 'contents',
+      '& tr': {
+        display: 'contents',
+        '& th': {
+          borderBottom: `2px solid ${AltinnAppTheme.altinnPalette.primary.blueMedium}`,
+        },
+      },
+    },
+    '& tbody': {
+      display: 'contents',
+      '& tr': {
+        display: 'contents',
+        borderBottom: `2px dotted ${AltinnAppTheme.altinnPalette.primary.blueMedium}`,
+        '& td': {
+          overflow: 'hidden',
+          textOverflow: 'ellipsis',
+        },
+      },
+    },
+  },
+  mobileTableRow: {
+    '& td': {
+      whiteSpace: 'nowrap',
+      textOverflow: 'ellipsis',
+      overflow: 'hidden',
+      alignItems: 'center',
+      display: 'flex',
+      alignSelf: 'stretch',
+      color: '#000',
+    },
+  },
+  fullGrid: {
+    gridColumn: '1 / -1',
+  },
+  tableHeader: {
+    borderBottom: `2px solid ${AltinnAppTheme.altinnPalette.primary.blueMedium}`,
+    '& tr': {
+      '& th': {
+        padding: '0px',
+        paddingLeft: '6px',
+        fontSize: '1.4rem',
+        fontFamily: 'Altinn-DIN',
+        fontWeight: '500 !important' as any,
+        color: '#000',
+      },
+    },
+  },
+  tableBody: {
+    '& tr': {
+      '& td': {
+        padding: '0px',
+        paddingLeft: '6px',
+        fontSize: '12px',
+        whiteSpace: 'nowrap',
+        textOverflow: 'ellipsis',
+        overflow: 'hidden',
+        fontFamily: 'Altinn-DIN',
+        fontWeight: 400,
+        borderBottom: `1px dotted ${AltinnAppTheme.altinnPalette.primary.blueMedium}`,
+        color: '#000',
+      },
+    },
+  },
+  editIcon: {
+    paddingLeft: '6px',
+    fontSize: '1.4rem !important',
+    fontWeight: '800 !important' as any,
+  },
+  editTextContainer: {
+    whiteSpace: 'nowrap',
+    textOverflow: 'ellipsis',
+    overflow: 'hidden',
+    color: '#000',
+    fontWeight: '500 !important' as any,
+    fontSize: '1.2rem',
+  },
+  textContainer: {
+    whiteSpace: 'nowrap',
+    textOverflow: 'ellipsis',
+    overflow: 'hidden',
+    color: '#000',
+    fontWeight: '500 !important' as any,
+    fontSize: '1.4rem',
+    minWidth: '0px',
+  },
+  editContainer: {
+    display: 'inline-block',
+    border: `2px dotted ${AltinnAppTheme.altinnPalette.primary.blueMedium}`,
+    padding: '12px',
+    width: '100%',
+    marginTop: '12px',
+    marginBottom: '12px',
+  },
+  deleteButton: {
+    padding: '0px',
+    color: 'black',
+    justifyContent: 'left',
+  },
+  delete: {
+    justifyContent: 'right',
+  },
+  select: {
+    fontSize: '1.6rem',
+    '&:focus': {
+      outline: `2px solid ${AltinnAppTheme.altinnPalette.primary.blueDark}`,
+    },
+  },
+});
+
+export const bytesInOneMB = 1048576;
+export const emptyArray = [];
+
+export function FileUploadWithTagComponent(props: IFileUploadWithTagProps): JSX.Element {
```

Comment:
This component reuses a lot of stuff from `FileUploadComponent`. I'm a little concerned that we end up having to maintain two separate components if we want to make changes in the style for instance. 
I would prefer if we could find a way to not duplicate shared styles at least (dropzone is now shared).

Reply:
Dragged dropzone as its own style and included in both fileuploaders.